x-pack/plugins/fleet/server/types/models/agent.ts

```diff
@@ -62,14 +62,25 @@ export const AgentEventSchema = schema.object({
   id: schema.string(),
 });
 
-export const NewAgentActionSchema = schema.object({
-  type: schema.oneOf([
-    schema.literal('POLICY_CHANGE'),
-    schema.literal('UNENROLL'),
-    schema.literal('UPGRADE'),
-    schema.literal('INTERNAL_POLICY_REASSIGN'),
-  ]),
-  data: schema.maybe(schema.any()),
-  ack_data: schema.maybe(schema.any()),
-  sent_at: schema.maybe(schema.string()),
```

Comment:
I removed `sent_at` and `ack_data` as there is no need for them when creating an action

Reply:
it's required in the saved_object we update that when the agent acknowledge an action, but it's not on a new action